jsapp/js/components/common/button.tsx

```diff
@@ -154,27 +154,39 @@ const Button = (props: ButtonProps) => {
   };
 
   return (
-    <button
-      className={classNames.join(' ')}
-      type={props.isSubmit ? 'submit' : 'button'}
-      aria-disabled={props.isDisabled}
-      onClick={handleClick}
-      onKeyUp={onKeyUp}
-      {...additionalButtonAttributes}
-    >
-      {props.startIcon && <Icon name={props.startIcon} size={iconSize} />}
-
-      {props.label && <span className='k-button__label'>{props.label}</span>}
-
-      {/* Ensures only one icon is being displayed.*/}
-      {!props.startIcon && props.endIcon && (
-        <Icon name={props.endIcon} size={iconSize} />
-      )}
-
-      {props.isPending && (
-        <Icon name='spinner' size={iconSize} classNames={['k-spin']} />
+    <>
+      {props.tooltip !== undefined && (
```

Comment:
I've noticed that if you don't add tooltip, this renders nothing. We need the button to be rendered with or without a tooltip :)

Reply:
I thought that somethign is wrong with Storybook, because it wasn't rendering anything for Button, so I've added a commit that improves `button.stories.tsx`. Please take a look at what it does when testing the tooltip:
<img width="184" alt="Screenshot 2023-12-13 at 15 19 35" src="https://github.com/kobotoolbox/kpi/assets/2521888/3b62cf31-2e57-413a-b7bf-c3a4e6cf99e7">